pattern-library/vitamin-t/public/vitamin-t-library/data.json

```diff
@@ -0,0 +1,141 @@
+{
+    "project_logo": "https://vitamintalent.com/contentAsset/image/c2449694-8cec-4359-9c4a-4d8b6c8c58d0/fileAsset/byInode/1/filter/Resize,Jpeg/resize_w/1200/jpeg_q/75/v/c2449694-8cec-4359-9c4a-4d8b6c8c58d0.jpg",
```

Comment:
Let's use a SVG logo instead of a JPG. There's one of each in the `svg` folder on A500.